src/platform-implementation-js/dom-driver/gmail/views/gmail-compose-view.ts

```diff
@@ -601,9 +602,19 @@ class GmailComposeView implements ComposeViewDriver {
     return this._destroyed;
   }
 
+  #getSubjectChangesStream() {
+    const subjectElement = this.getSubjectInput();
+    return Kefir.fromEvents(subjectElement, 'input').map(() => {
```

Comment:
I'm a little worried about using `'input'` events. I did attempt to get #911  to work with `makeMutationObserverStream`. Events only seemed to fire on the first character input though.

Reply:
Mutation observers are good for content editables; input events is best input elements like the subject field.